docs/user/dead_time_correction.rst

```diff
@@ -0,0 +1,111 @@
+.. _dead_time_correction:
+
+SingleReadoutDeadTimeCorrection
+===============================
+
+Dead time is the time after an event that a detector is not able to detect another event.
+For a paralyzable detector, an event that happens during the dead time restarts the dead time. For
+a non-paralyzable detector, the event is simply lost and does not cause additional dead time.
+
+Dead-time correction corrects for detector dead time by weighing the events according to:
+
+.. math:: N = M \frac{1}{(1-\mathrm{rate} \cdot (\frac{t_{\mathrm{dead}}}{t_{\mathrm{bin}}}))}
+
+for non-paralyzable detectors and
+
+.. math:: N = M \frac{\mathrm{Re} (\mathrm{W}(-\mathrm{rate} \cdot (\frac{t_{\mathrm{dead}}}{t_{\mathrm{bin}}})) )}{\frac{\mathrm{rate}}{t_{\mathrm{bin}}}}
+
+for paralyzable detectors, where
+
+| :math:`N` = true count
+| :math:`M` = measured count
+| :math:`t_{\mathrm{dead}}` = dead time
+| :math:`t_{\mathrm{bin}}` = TOF bin width
+| :math:`\mathrm{rate}` = measured count rate
+| :math:`\mathrm{W}` = Lambert W function
```

Comment:
can you  associate link https://en.wikipedia.org/wiki/Lambert_W_function to string `Lambert W function`, please?